src/extensions/default/AutoUpdate/main.js

```diff
@@ -594,6 +643,7 @@ define(function (require, exports, module) {
      */
     function showStatusInfo(statusObj) {
         if (statusObj.target === "initial-download") {
+            HealthLogger.sendAnalyticsData(AUTOUPDATE_DOWNLOAD_START, "autoUpdate", "download", "started", "");
```

Comment:
@niteskum this piece of code is called multiple times, while the download is in progress. Is the event logged only once, or each time this function is called?

Reply:
Event will be logged only once